StoreSpidLogs/index.ts

```diff
@@ -0,0 +1,83 @@
+import { Context } from "@azure/functions";
+import { AzureContextTransport } from "io-functions-commons/dist/src/utils/logging";
+import * as t from "io-ts";
+import { UTCISODateFromString } from "italia-ts-commons/lib/dates";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { IPString, PatternString } from "italia-ts-commons/lib/strings";
+import * as winston from "winston";
+
+/**
+ * This type wraps a Spid Request/Response message sent over an azure storage queue, namely "spidmsgitems"

```

Comment:
```suggestion
 * This type wraps a Spid Request/Response message sent over an azure storage queue

```

try to not put to may implmentation details into comments as they're subject to refactoring (ie. naming)